### PR TITLE
Agreements

### PR DESCRIPTION
**Pull request by corsairmarks**
Agreements, specialist subject perks, and, all the triggers/effects I missed on the first go-round (most of which deal with subjects and/or agreements).

### CODE REVIEW

File: config/common/agreements.cwt [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+	}
+	## cardinality = 0..1
+	## replace_scope = { this = country root = country }
+	### A modifier that applies to the Agreement owner country. Can stack multiple times.
+	owner_modifier = {
+		alias_name[modifier] = alias_match_left[modifier]
+ 	}
+	## cardinality = 0..1
+	## replace_scope = { this = country root = country }
+	### A modifier that applies to the Agreement owner country. Can stack multiple times.
```
Comment (CaligulaCaesar) [inline config/common/agreements.cwt:157]:
```suggestion
	### A modifier that applies to the Agreement owner country. Only applies once.
```

File: config/common/agreements.cwt [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+	unique_owner_modifier = {
+		alias_name[modifier] = alias_match_left[modifier]
+ 	}
+	## cardinality = 0..1
+	## replace_scope = { this = country root = country }
+	### A modifier that applies to the Agreement owner country. Only applies once.
```
Comment (CaligulaCaesar) [inline config/common/agreements.cwt:163]:
```suggestion
	### A modifier that applies to the Agreement target country. Can stack multiple times.
```

File: config/effects.cwt [hunk 3]
```diff
@@ -3464,55 +3469,129 @@ alias[effect:start_situation] = {
...
+alias[effect:remove_holding] = {
+	holding = any
+	holding = none
+	holding = <building.corporate>
+	holding = <building.holding>
+	owner = scope_group[target_country]
+}
+###Begins construction of a specific holding on the scoped planet
+alias[effect:add_holding] = {
+	holding = any
```
Comment (CaligulaCaesar) [inline config/effects.cwt:3514]: I'm pretty sure you need to specify which one, not any or none

File: config/common/agreements.cwt [hunk 4]
```diff
@@ -154,7 +154,7 @@ agreement_term_value = {
  	}
 	## cardinality = 0..1
 	## replace_scope = { this = country root = country }
-	### A modifier that applies to the Agreement owner country. Can stack multiple times.
+	### A modifier that applies to the Agreement owner country. Only applies once.
```
Comment (corsairmarks) [inline config/common/agreements.cwt:157]: Good catch on my copy/paste errors

**Pull request merged**